op-node/rollup/derive/channel_bank.go

```diff
@@ -60,14 +60,21 @@ func (cb *ChannelBank) Origin() eth.L1BlockRef {
 	return cb.prev.Origin()
 }
 
+func (cb *ChannelBank) maxSize() int {
+	if cb.Config.IsFjord(cb.Origin().Time) {
+		return MaxChannelBankSizeV1
+	}
+	return MaxChannelBankSize
+}
```

Comment:
Wdyt about starting to treat the `rollup.Config` more like a specs object and add this logic instead to it? Then we have the protocol rules encapsulated with the rollup config, not as methods on various structs.

```go
func (c *Config) MaxChannelBankSize(t uint64) int {
	if c.IsFjord(t) {
		return maxChannelBankSizeV1 // or ...Ecotone
	}
	return maxChannelBankSizeV0 // or ...Bedrock
}
```